src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -181,7 +186,8 @@ class LocalContextScenarioTest(ScenarioTest):
     def __init__(self, method_name, config_file=None, recording_name=None, recording_processors=None,
                  replay_processors=None, recording_patches=None, replay_patches=None, working_dir=None):
         super(LocalContextScenarioTest, self).__init__(method_name, config_file, recording_name, recording_processors,
-                                                       replay_processors, recording_patches, replay_patches)
+                                                       replay_processors, recording_patches, replay_patches,
+                                                       random_config_dir=True)
```

Comment:
`ConfigureGlobalDefaultsTest` runs `config param-persist off` in `tearDown`. I think use `random_config_dir` is necessary

Reply:
Local Context/Param Persist has been deprecated. Consider disable the test instead: https://github.com/Azure/azure-cli/issues/24726